docs-v2/components/homepage/Resources.tsx

```diff
@@ -0,0 +1,28 @@
+import { Pane, majorScale, Heading, Paragraph, Link, useTheme } from "evergreen-ui"
+import React from "react"
+import ResourceCard from "./ResourceCard"
+
+const Resources = () => {
+  const { colors } = useTheme()
+
+  return (
+    <Pane background={colors.gray50}>
+      <Pane
+        paddingY={majorScale(5)}
+        marginX="auto"
+        maxWidth={majorScale(143)}
```

Comment:
Let's figure out a consistent maxWidth and put that in a common `layout-constants` file, so pages don't feel jumpy in between.